xknx/telegram/address.py

```diff
@@ -81,29 +80,29 @@ class PhysicalAddress(BaseAddress):
         r"^(?P<area>\d{1,2})\.(?P<main>\d{1,2})\.(?P<line>\d{1,3})$"
     )
 
-    def __init__(self, address):
+    def __init__(self, address: Union[object, str, int, tuple]) -> None:
```

Comment:
We can't use the same Type as the current class unfortunetly :/

Reply:
this is our excuse to drop 3.6
https://stackoverflow.com/a/49872353/4516732
https://www.python.org/dev/peps/pep-0563/